addons/payment/data/payment_icon_data.xml

```diff
@@ -60,11 +80,6 @@
         <field name="image" type="base64" file="payment/static/img/western_union.png"/>
     </record>
 
-    <record id="payment_icon_cc_ideal" model="payment.icon">
-        <field name="name">iDEAL</field>
-        <field name="image" type="base64" file="payment/static/img/ideal.png"/>
-    </record>
-
```

Comment:
Why moving this elsewhere?

Reply:
So that I had all local payment methods grouped.

Reply:
That's a useless diff for a stable version. We could eventually do that in master when updating this file if it makes sense for all acquirers, not just Stripe, but not in this fix.

Reply:
moved back